#431 Задано наименование предопределенного регламентного задания (#786)
1C-Company/v8-code-style: bundles/com.e1c.v8codestyle.md/markdown/mdo-scheduled-job-description.md

# The description of the predefine sheduled job is set

> Predefined Sheduled jobs should not
> be description in the configuration. Instead, it is enough to set a synonym for a
> predefined sheduled job.

Reviewer: ```suggestion
Description of predefined Scheduled jobs should not be set. 
Instead, it is enough to set only synonym for a predefined scheduled job.
```
